podcasts/softwerkercast.yml

```
@@ -3,4 +3,10 @@ website: https://www.codecentric.de/softwerkercast
 podcastIndexID: 4530860
 rssFeed: https://softwerkercast.podigee.io/feed/mp3
 spotify: https://open.spotify.com/show/0GMFlDJabOtXYm8u8HfAvL
-description: "Der SoftwerkerCast, unser neues Podcast-Format, präsentiert euch Softwerker-Wissen zum Reinhören und Nachhören. Wie auch bei den anderen Softwerker-Formaten erwartet euch im SoftwerkerCast das volle Themenspektrum aus der bunten Welt der Softwareentwicklung: von A wie APM bis Z wie Zero Downtime. Paradigmen und neue Trends, Evergreens und Eintagsfliegen."
+description: "Der SoftwerkerCast präsentiert euch Softwerker-Wissen zum Reinhören und Nachhören. Wie auch bei den anderen Softwerker-Formaten erwartet euch im SoftwerkerCast das volle Themenspektrum aus der bunten Welt der Softwareentwicklung: von A wie APM bis Z wie Zero Downtime. Paradigmen und neue Trends, Evergreens und Eintagsfliegen."
+tags:
+    - Softwareentwicklung
+    - Software Development
+    - Cloud
+    - IT Security
+    - Platform Engineering
```

```suggestion
    - Platform Engineering

```

Adding a newline for yamlint